packages/plugin-table/src/TableRowObjectDomRenderer.ts

```diff
@@ -0,0 +1,23 @@
+import { AbstractRenderer } from '../../plugin-renderer/src/AbstractRenderer';
+import { TableRowNode } from './TableRowNode';
+import { ObjectDomRenderingEngine } from '../../plugin-html/src/ObjectDomRenderingEngine';
+import { DomObject } from '../../plugin-html/src/DomObjectMap';
+import { Attributes } from '../../plugin-xml/src/Attributes';
+
+export class TableRowObjectDomRenderer extends AbstractRenderer<DomObject> {
+    static id = ObjectDomRenderingEngine.id;
+    engine: ObjectDomRenderingEngine;
+    predicate = TableRowNode;
+
+    /**
+     * Render the TableRowNode along with its contents.
+     */
+    async render(row: TableRowNode): Promise<DomObject> {
+        const objectRow: DomObject = {
+            nodeName: 'TR',
+            children: row.children(node => node.tangible),
```

Comment:
All `children()` are `tangible`, that's the point of `children()` vs `childVNodes`.